packages/common/src/search/utils.ts

```diff
@@ -349,3 +349,25 @@ export function getResultSiteRelativeLink(
   }
   return siteRelativeLink;
 }
+
+/**
+ * Adds default predicates for item target entity
+ *
+ * @param query IQuery to search items
+ * @returns an IQuery with default predicates
+ */
+export function getItemQueryWithDefaultPredicates(query: IQuery): IQuery {
```

Comment:
Looking great! Since you are modifying an existing query (albeit via cloning) as part of this function, I would rename the function to `addDefaultItemSearchPredicates` and specify in the TS DOC comment that a clone is returned.

Reply:
I agree! changed the name to `addDefaultItemSearchPredicates` and added the comment.